SimPPS/RPDigiProducer/src/RPLinearChargeDivider.cc

```diff
@@ -0,0 +1,142 @@
+#include "SimPPS/RPDigiProducer/interface/RPLinearChargeDivider.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include "DataFormats/GeometryVector/interface/LocalVector.h"
+#include "Geometry/VeryForwardRPTopology/interface/RPTopology.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+RPLinearChargeDivider::RPLinearChargeDivider(const edm::ParameterSet &params,  CLHEP::HepRandomEngine& eng,
+    RPDetId det_id) : params_(params), rndEngine(eng) , _det_id(det_id)
+{
+  verbosity_ = params.getParameter<int>("RPVerbosity");
+
+  fluctuate = new SiG4UniversalFluctuation();
```

Comment:
Can this be a regular member instead of pointer?

Reply:
@mundim , "fluctuate" should be "fluctuate_", it may be also defined by value or std::unique_ptr